_posts/2016/03/2016-03-01-venn-diagram.md

```diff
@@ -0,0 +1,83 @@
+circles for Lessons, Learners and Instructors. I've tried to use this diagram to
+build my own mental model for how what we do in terms of core activities can
+scale and grow. Through this process I've come to think of the structure in this
+diagram as  podular, or a fractal element that repeats at various community
+scales such as university, research network, nation or worldwide.
+
+![The Components of Software Carpentry](/files/2016/03/venn_diagram.png)
+
+At each of our partner organizations we may have lessons, instructors and
+learners with their own unique and local perspectives, working toward impactful
+workshops that are appropriate for their own community. Internationally, we are
+working toward spreading consensus lessons and ethos of using open source tools
+for open and collaborative science to scientific communities via our workshops
+while using evidence based teaching methods.
+
+These workshops sit at the nexus of the diagram and in many ways showcase, in a
+focused event, what it is that we stand for. They reflect the impacts we would
+like to have on changing how science is thought about in the context of
+computation. The brands "Software Carpentry" and "Data Carpentry" reflect a
+particular set of opinionated lessons arranged to have specific impacts on
+learners. This is why we work so hard to make sure that a workshop called
+"Software Carpentry" or "Data Carpentry" is being taught using our methods (the
+instructor is "badged") and with instructors who have studied the
+community-developed lessons.
+
+Locally when we deliver a workshop we're working to bring together lessons
+learners, and instructors that can deliver impactful workshops. To be prepared
+for this we strive to convince learners that spending two days with
+these lessons and our instructors will be impactful and helpful to how they
+work. When you put this all together, we're not just developing training and
+delivering workshops, we as a community collectively own the lessons and are
+advancing, testing and refining the evidence based teaching best practices that
+we share with the uninitiated and reinforce through our instructor training.
+
+Outside of our core lessons and flagship workshops, you are strongly
+encouraged to duplicate this structure and apply it at smaller scales toward
+the specific needs of your own communities. This **is** how we grow and test
+new  ideas and lessons. As you do this we want to know what you've learned, we
+want to hear your success stories, we want to hear about your spectacular
+failures. Overall, we want The Carpentries together to be a community where the
+most broadly applicable lessons pertaining to the tools and best practices
+needed to do modern research can come to be curated and improved together. At
+the same time, we have [thriving global
+conversations](http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/)
```

Comment:
`{{site.mailing_lists}}/pipermail/discuss_lists.software-carpentry.org` - if we ever move the mailing list host, this'll help us find instances.